contracts/interfaces/IAxelarGasService.sol

```diff
@@ -216,45 +244,100 @@ interface IAxelarGasService is IUpgradable {
+     * @param txHash The transaction hash of the cross-chain call
+     * @param logIndex The log index for the cross-chain call
+     * @param refundAddress The address where refunds, if any, should be sent
+     */
     function addNativeGas(
         bytes32 txHash,
         uint256 logIndex,
         address refundAddress
     ) external payable;
 
+    /**
+     * @notice Add additional gas payment using ERC20 tokens after initiating an express cross-chain call.
+     * @dev This function can be called on the source chain after calling the gateway to express execute a remote contract.
+     * @param txHash The transaction hash of the cross-chain call
+     * @param logIndex The log index for the cross-chain call
+     * @param gasToken The ERC20 token address used to add gas
+     * @param gasFeeAmount The amount of tokens to add as gas
+     * @param refundAddress The address where refunds, if any, should be sent
+     */
     function addExpressGas(
         bytes32 txHash,
-        uint256 txIndex,
+        uint256 logIndex,
         address gasToken,
         uint256 gasFeeAmount,
         address refundAddress
     ) external;
 
+    /**
+     * @notice Add additional gas payment using native currency after initiating an express cross-chain call.
+     * @dev This function can be called on the source chain after calling the gateway to express execute a remote contract.
+     * @param txHash The transaction hash of the cross-chain call
+     * @param logIndex The log index for the cross-chain call
+     * @param refundAddress The address where refunds, if any, should be sent
+     */
     function addNativeExpressGas(
         bytes32 txHash,
         uint256 logIndex,
         address refundAddress
     ) external payable;
 
+    /**
+     * @notice Allows the gasCollector to collect accumulated fees from the contract.
+     * @dev Use address(0) as the token address for native currency.
+     * @param receiver The address to receive the collected fees
+     * @param tokens Array of token addresses to be collected
+     * @param amounts Array of amounts to be collected for each respective token address
+     */
     function collectFees(
         address payable receiver,
         address[] calldata tokens,
         uint256[] calldata amounts
     ) external;
 
+    /**
+     * @notice Refunds gas payment to the receiver in relation to a specific cross-chain transaction.
+     * @dev Only callable by the gasCollector.
+     * @dev Use address(0) as the token address to refund native currency.
+     * @param txHash The transaction hash of the cross-chain call
+     * @param logIndex The log index for the cross-chain call
+     * @param receiver The address to receive the refund
+     * @param token The token address to be refunded
+     * @param amount The amount to refund
+     */
     function refund(
+        bytes32 txHash,
+        uint256 logIndex,
```

Comment:
I changed this to the function definition of the non-deprecated `refund` function, not sure if it was kept as the deprecated function definition intentionally for backward compatibility or something

Reply:
ok, should be fine